Gemfile

```diff
@@ -101,7 +101,7 @@ group(:omnibus_package) do
   gem "chef-sugar"
   gem "mixlib-versioning"
   gem "artifactory"
-  gem "opscode-pushy-client", ">= 2.1"
+  gem "opscode-pushy-client", git: "https://github.com/chef/opscode-pushy-client.git", branch: "master"
```

Comment:
that'll almost certainly break appbundling

Reply:
(actually i should stop saying that -- bundle install won't install it correctly into the gem library first)

Reply:
👍

Reply:
Switched to the pre gem so we're good to go now